Address feedback on the intro (#393)
cfrg/draft-irtf-cfrg-vdaf: draft-irtf-cfrg-vdaf.md

from the literature.

* The Prio system {{CGB17}} allows for the privacy-preserving computation of a
  variety aggregate statistics, combining additive secret sharing as described
  above with a mechanism for checking the validity of each measurement. In
  {{prio3}} we specify Prio3, a VDAF that follows the same overall framework as
  the original Prio protocol, but incorporates techniques introduced in
  {{BBCGGI19}} that result in significant performance gains.

* The Poplar protocol {{BBCGGI21}} solves the heavy-hitters problem in a
  privacy-preserving manner. Here each client holds a bit-string, and the goal
  of the aggregation servers is to compute the set of strings that occur at
  least `t` times form some threshold `t`. The core primitive in their protocol

>   is a secret sharing of a point function {{GI14}} (`g` in the notation above)

Author: The operative word is "allows", as in "they evaluate point function shares on each of their input shares and add up the output shares to get share of the count". Is this is clear enough or do we need to fix it?

>   is a secret sharing of a point function {{GI14}} (`g` in the notation above)

Reviewer: I think it's fine as-is

Prio3 to precomputation attacks, which the larger field mitigates. (See
{{DPRS23}}, Theorem 1.) Note that privacy is not susceptible to such attacks.

> Another way to mitigate this issue (or improve robustness in general) is to:wq

Author: 😬

`PROOFS == 2` is feasible, if impractical; but `PROOFS == 1` is completely
broken for such a small field.

## Choosing the Number of Aggregators {#num-aggregators}

Two Aggregators are required for privacy in our threat model, but some (V)DAFs,
including Prio3 ({{prio3}}), allow for any number of Aggregators, only one of
which needs to be trusted in order for the computation to be private. To hedge
against corruptions that happen during the course of the attack, deployments
may consider involving more than two Aggregators as described for example in

> {{star-topo}}. Note however that not all schemes, including Poplar1, are
> compatible with this mode of operation.

Reviewer: I found "including" to be confusing here, so I rephrased it.
```suggestion
{{star-topo}}. Note however that some schemes are not compatible with this mode of operation,
such as Poplar1.
```
